applications/webapp/src/Pages/User/userPage.tsx

```diff
@@ -10,65 +10,88 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from "react";
+import React, { useState, useMemo } from "react";
 import { Box, useTheme } from "@mui/material";
 import ProfileHolder from "./components/profileHolder";
 import DetailHolder from "./components/detailHolder";
-import { styles } from "./styles/style";
+import { EditableCardElevation, styles } from "./util/style";
 import { useQuery } from "@apollo/client";
 import { getUserQuery } from "./graphql/userQuery";
+import { CardContext, UserContext } from "./context";
 
 export default function UserPage() {
   const theme = useTheme();
   const style = styles(theme).userPage;
 
-  const mockUser = {
-    phonenumber: "123456789",
-    address: "Addis Ababa",
-    birthDate: "April 19 2001",
-    description: "Hi I am Yonas",
-  };
+  const [editable, setEditable] = useState(true);
+  const [elevation, setElevation] = useState(0);
 
   const { data } = useQuery(getUserQuery);
```

Comment:
Doing it this way you subject yourself to endlessly repeating same things over and over again. Overly verbose and not DRY. Consider creating a custom generic hook to return typed results based on provided query name and variables. This change is to big for this PR so this comment is for awareness. I will add new story for this custom hook